rocky/reports/report_types/aggregate_organisation_report/appendix.html

```diff
@@ -5,7 +5,7 @@
         <h2 id="appendix">{% translate "Appendix" %}</h2>
         {% include "aggregate_organisation_report/term_overview.html" %}
 
-        <h3 id="selected-oois">{% translate "Selected objects" %}</h3>
+        <h3 id="selected-oois">{% translate "Selected objects" %} {{ total_oois }}</h3>
```

Comment:
Would be nice if this was also pluralized, but maybe not necessary. I'll leave it up to you.

Reply:
I think I will leave this one, we need to refactor later more of these type of things
Selected objects (1) will be good for now.
I will add the brackets though